Shared/Models/CampusEatery.swift

```diff
@@ -205,6 +211,14 @@ struct CampusEatery: Eatery, Codable, DefaultsSerializable {
             self.diningMenu = nil
         }
 
+        if let expandMenu = expandedMenu {
+            self.expandedMenu = ExpandedMenu(data: expandMenu)
+        } else {
+            self.expandedMenu = nil
```

Comment:
shouldn't expandedMenu already be nil if it skipped the if let?

Reply:
Since this is in the initializer of a struct, it actually requires that I include `self.expandedMenu = nil` or else it gives a syntax error saying `Return from initializer without initializing all stored properties`